src/entrypoints/FieldAddon/FieldAddon.tsx

```diff
@@ -188,6 +188,11 @@ export default function FieldAddon({ ctx }: Props) {
     ctx.setHeight(0)
     return <></>
   }
+  
+  if (locales.length <= 1) {
```

Comment:
I would actually prefer this to be used in the check above. And not repeat this:
```javascript
ctx.setHeight
return <></>
```

So I would put in on line 187 and have the check be
`((fieldHasFieldValue(fieldValue, ctx) && !isDefaultLocale) || locales.length <= 1)`